app/javascript/ra/events/EventForm.vue

```
@@ -462,6 +462,18 @@
         );
         return newTime;
       },
+
+      minDate() {
+        return process.env.DATES_REGIONAL_PITCH_EVENTS_BEGINS_YEAR + '-' +
+          process.env.DATES_REGIONAL_PITCH_EVENTS_BEGINS_MONTH + '-' +
+          process.env.DATES_REGIONAL_PITCH_EVENTS_BEGINS_DAY
```

I tried string interpolation here, but (imho) it didn't make it any more readable. 😕 Open to suggestions!

---

Having one environment variable contain the entire date would probably help, but I was just following the convention of our other environment variables where the date is split up. 💇‍♂ 💅 💇 🤷‍♂ 😺

---

I had absolutely no idea you could use `process.env` from front end code, so this is interesting. 

I was assuming we'd read the config [somewhere in here](https://github.com/Iridescent-CM/technovation-app/blob/qa/app/constants/important_dates.rb) and render it as a data attribute onto a page or whatever it is this Vue app otherwise does to get initialized. (Note, I'd prefer that Vue apps initialize with an initial ajax call but many of them don't currently.)

This is in a way much cleaner, but at the cost of decentralizing some of our reading-dates-from-the-env setup. I guess my question is whether or not this will "fail fast" if that config is missing? Or will the front end just display a broken date picker? What if instead of string concatenation you pass the variables to `new Date(...)`, would that fail fast if the config wasn't there, or will the app still run with a broken date picker?